src/main/java/frc/team2412/robot/util/AutonomousChooser.java

```diff
@@ -0,0 +1,62 @@
+package frc.team2412.robot.util;
+
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.team2412.robot.Subsystems;
+import frc.team2412.robot.commands.autonomous.Follow2910TrajectoryCommand;
+
+public class AutonomousChooser {
+    private AutonomousTrajectories trajectories;
+
+    private SendableChooser<AutonomousMode> autonomousModeChooser = new SendableChooser<>();
+
+    public AutonomousChooser(AutonomousTrajectories trajectories) {
+        this.trajectories = trajectories;
+
+        autonomousModeChooser.setDefaultOption("Square Path Auto", AutonomousMode.SQUARE_PATH_AUTO);
+        autonomousModeChooser.addOption("Star Path Auto", AutonomousMode.STAR_PATH_AUTO);
```

Comment:
All the path-related information (name, really ought to be part of an AutonomousTrajectory class, so it can simply be enumerated & read out of the object rather than having to be hardcoded several places (here, the switch() in getCommand(), and the get*AutoCommand methods). But that can happen for v2.